pkg/schemaconv/testdata/swagger.json

```diff
@@ -78578,6 +78578,10 @@
    },
    "io.k8s.api.apps.v1beta1.DeploymentStrategy": {
     "description": "DeploymentStrategy describes how to replace existing pods with new ones.",
+    "x-kubernetes-oneof-discriminator": "type",
+    "x-kubernetes-oneof-fields": {
+     "rollingUpdate": "RollingUpdate"
```

Comment:
I assume this is a map from field name to discriminator value? Can we change the name of "x-kubernetes-oneof-fields" until this is clearer?

Reply:
+1

Reply:
Changed to `x-kubernetes-unino-fields-discriminated`